src/frontend/src/binder/expr/function.rs

```diff
@@ -1426,6 +1426,8 @@ impl Binder {
                 ("pg_is_in_recovery", raw_literal(ExprImpl::literal_bool(false))),
                 // internal
                 ("rw_vnode", raw_call(ExprType::Vnode)),
+                #[cfg(debug_assertions)] // only available for testing purposes
```

Comment:
A mind burden to me. I think it's not bad to include it in official release. Actually might be a handy way for users to test whether the license key works.

```suggestion
```